src/lib/InstantSearch.ts

```diff
@@ -245,23 +246,22 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
   }
 
   /**
-   * Adds a widget. This can be done before and after InstantSearch has been started. Adding a
-   * widget after InstantSearch started is considered **EXPERIMENTAL** and therefore
-   * it is possibly buggy, if you find anything please
-   * [open an issue](https://github.com/algolia/instantsearch.js/issues/new?title=Problem%20with%20hot%20addWidget).
-   * @param  widget The widget to add to InstantSearch. Widgets are simple objects
-   * that have methods that map the search life cycle in a UI perspective. Usually widgets are
-   * created by [widget factories](widgets.html) like the one provided with InstantSearch.js.
+   * Adds a widget. This can be done before and after InstantSearch has been started.
```

Comment:
```suggestion
   * Adds a widget to the search instance.
   * A widget can be added either before or after InstantSearch has started.
```

We could even add more information here. Example:

- Adding a widget sets the according search parameters to communicate with the server.
- A widget is responsible for setting and reading the UI state.

Reply:
I'm not sure people can even see these comments, let's not get in a discussion totally changing those just because I removed an "experimental" warning